src/field_theory/intermediate_field.lean

```diff
@@ -273,6 +273,8 @@ def map (f : L →ₐ[K] L') : intermediate_field K L' :=
   neg_mem' := λ x hx, (S.to_subalgebra.map f).neg_mem hx,
   .. S.to_subalgebra.map f}
 
+@[simp] lemma coe_map (f : L →ₐ[K] L') : (S.map f : set L') = f '' S := rfl
```

Comment:
Do we have this lemma for submodules etc already? If not, I think we should add them all at the same time, as that's a good way to verify that the lemma is safe as simp

Reply:
It's `submodule.map_coe`, `subgroup.coe_map`, `submonoid.coe_map`.

Reply:
All of which are `simp`?

Reply:
Yes.